### PR TITLE
Automating Strapi workflow for dev deployments

### PR DESCRIPTION
**Pull request by parulmishra**
Automating Strapi workflow for dev deployments

### CODE REVIEW

File: ansible/tasks/check_strapi_image_exists-test-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+  - name: Fetch imagestream {{ react_infra_name }}:{{ version }}
```
Comment (thegentlemanphysicist) [inline ansible/tasks/check_strapi_image_exists-test-prod.yaml:2]: I believe this should be the strapi infraname, unless the two are always the same

File: ansible/tasks/check_strapi_image_exists-test-prod.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    set_fact: 
+      imagestreamtag: "{{ lookup('k8s', kind='ImageStreamTag', namespace=image_namespace, resource_name=strapi_infra_name + ':' + version) }}"
+    until: (imagestreamtag|length > 0)
+    delay: 60
+    retries: 20
+    ignore_errors: True
+  - name: Image Tag {{ version }} does not exist in {{ image_namespace }}. 
+    debug: 
+      msg: "Image Tag {{ version }} does not exist in {{ image_namespace }}. \
+           This is because either you have not run ansible-playbook build-react.yaml -e PR={{ PR }} \
```
Comment (thegentlemanphysicist) [inline ansible/tasks/check_strapi_image_exists-test-prod.yaml:12]: This message should be a referencing a different playbook

**Review by thegentlemanphysicist — COMMENTED** [review]
Found a couple of references to react in the strapi infra-code.

File: ansible/tasks/check_strapi_image_exists-test-prod.yaml (hunk 1 above)
Comment (parulmishra) [reply ansible/tasks/check_strapi_image_exists-test-prod.yaml:2]: Good catch. Just the step name, doesn't affect execution. But I will change it.

File: ansible/tasks/check_strapi_image_exists-test-prod.yaml (hunk 2 above)
Comment (parulmishra) [reply ansible/tasks/check_strapi_image_exists-test-prod.yaml:12]: Yes, forgot to update the debug message. Will update.

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**